test/services/TwitterServiceTest.ts

```diff
@@ -67,7 +68,9 @@ describe("TwitterService", () => {
 
 			await twitterService.handleTwitterStream({
 				id_str: "",
-				text: "@This does not start with an @"
+				extended_tweet: {
+					full_text: "@This does not start with an @"
```

Comment:
This is totally my fault, but the message doesn't make sense... it does begin with an "@".